.travis.yml

```diff
@@ -69,6 +74,7 @@ install:
   - cd ../..
 
 script:
+  - pycodestyle --show-source --show-pep8 $PYCODESTYLE_CONVENTIONS $CGAL_PYTHON_EXAMPLES || pycodestyle --statistics $PYCODESTYLE_CONVENTIONS -qq $CGAL_PYTHON_EXAMPLES
```

Comment:
could you explain what this line will do? Will we get a travis failure each time an example is not well formatted?

Reply:
and more importantly is the error displayed by travis obvious enough for the developer?

Reply:
Yes and yes. You can take a look at the failed ones above.
What you get is even a full description of the coding rule and examples how it should be written instead. The second part of the line after the OR clause will just give stats about the found errors.